Automattic/wp-calypso
# Update/videopress remove tube

#### Proposed Changes

* remove `tube` as a suggested suffix for wordpress domains in the VideoPress onboarding flow

#### Testing Instructions



* start the videopress onboarding flow at `/start/videopress`
* type in a site name, watch the domain suggestions
* the domain suggestions should behave as it did before but it should never suggest a `tube` suffix

#### Pre-merge Checklist






Related to # 1284-gh-Automattic/greenhouse
